readme.md

```diff
@@ -74,7 +80,12 @@ tags: [
 ```
 
 #### Multiline String
-Multiline string is a string that spans multiple lines. Its enclosed in parenthesis `()`.
+Multiline string is a string that spans multiple lines, enclosed in parentheses (`()`). The
+content *must* begin on the line after the opening parenthesis (`(`) and the
+closing parenthesis (`)`) *must* be on the line after the end of the multiline
+string. Multiline strings are indented at least two spaces more than the key
+which contains them. This indentation is removed on parsing.
```

Comment:
If it is possible to do, using a more typical multiline character approach (`'''` or `"""` delimited like python, elixir, or hjson) would be better than parentheses. However, I added much more explicit definition on where the text begins (the line *after* opening parenthesis) and ends (the line *before* closing parenthesis) and what happens to the indented text.

Is the indented text *always stripped at `key-indent + 2` or is it `first non-whitespace of first line`?

That is, is:

```
(
  xyz
)
```

the same as:

```
(
      xyz
)
```

It may be desirable to add an *explicit* optional boundary character if there should be a difference:

```
(
      |   xyz
)
```

In this case, the value would be `"   xyz"`.

Also unclear: does the *last* line of a multiline string include a newline? That is, if I want to have `abc\ndef\n`, do I do:

```
(
  abc
  def
)
```

or

```
(
  abc
  def

)
```

Reply:
So based on my reading the delimter format was `"""` was introduced primarily in bash shells, and has been adopted across programming languages.

There has been confusion around how new line gets treated.
For instance, TOML assumes the sentence begins in the next line after `'''`, but does not trim the last line

#### TOML
`text`
```text
abc
def
```

`toml`
```toml
data = '''
abc
def'''
```

not so clean if you don't want your data to have a new line

#### YAML
YAML is even more confusing and there appears to be [9 ways](https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines) of how to represent multiline strings 

#### Bru
With Bru we have an opportunity to make this very simple.
You put the rules very succinctly
* content *must* begin on the line after the opening parenthesis (`(`) and the
closing parenthesis (`)`) *must* be on the line after the end of the multiline
string.
* Multiline strings are indented *two spaces* more than the key
which contains them. This indentation is removed on parsing.

#### Some examples in Bru
`text`
```text
line 1
line 2
```

`bru`
```groovy
data: (
  line 1
  ine 2
)
```

The below example has a newline before and after
`text`
```text

  line 1 with 2 spaces before
line 2
     line 3 with 4 spaces before

```

`bru`
```groovy
data: (
  
    line 1 with 2 spaces before
  line 2
      line 3 with 4 spaces before
  
)
```

On the choice of `()` as delimiters, I feel this makes the format a tad bit less noisy and more elegant.
IMO, we don't need three `'''` quotes to delimit a multiline string when we the language is indentation based and specifies clearly how the multiline string gets parsed.

It also plays well in terms of consistency of having brackets as delimters for all the composite types
* multimap `{}`
* array - `[]`
* multistring - `()`

We can discuss further on this in the call.

Reply:
As far as I know, bash does not support `'''` or `"""` as special forms. Python would be the language that I am aware of that introduced this, and they have always been multi-line.

Wording the indentation spec here is going to be subtle, because not all multi-lines will be attached to keys. I would reword that to something like:

> Multi-line strings are enclosed with `(` and `)`, which must be on separate lines from the content of the multi-line string. The content must be indented two spaces *more* than the beginning of the line where the opening parenthesis begins.
>
> Examples:
>
> ```groovy
> key: (
>   content starts here
> )
> ```
>
> ```groovy
> [
>   (
>     content starts here
>   )
> ]

I realize that interoperability with other tools is not a primary goal, but if the spec is clear enough to allow the implementation of non-Bruno parsers, especially in other languages, then there will be some comfort that the spec is robust.